doc/user/README.md

```diff
@@ -62,26 +60,24 @@ Railroad diagrams consist of two parts:
 
 - `bnf` files that describe the grammar you want to convey in a railroad
   diagram.
-- `html` files that contain an `svg` generated by
+- `svg` files that contain an `svg` generated by
   <https://www.bottlecaps.de/rr/ui>
 
 #### To generate diagrams
 
-1. Modify/create `bnf` files in `layouts/partials/sql-grammar/bnf`.
-1. Ensure your diagram looks like you want using
+1. Modify the grammar in `sql-grammar/sql-grammar.bnf`, adding new nonterminals
+   as necessary.
+1. Ensure the diagram looks like you want using
    <https://www.bottlecaps.de/rr/ui>.
-2. Go to `util/rr-diagram-gen/`
-3. Run `make run`.
+2. Run `sql-grammar/generate.sh`.
 
 You can now include the diagram on a page using:
 
 ```html
-{{&lt; diagram "SOME_DIAGRAM.html" &gt;}}
+{{< diagram "SOME-NONTERMINAL.svg" >}}
```

Comment:
Man; I tried for like 15 minutes one day to get this to work and it just absolutely refused. Does the markdown actually render this as a code block?

Reply:
I think so! https://github.com/MaterializeInc/materialize/blob/4023f7d6ce771c8961fdc8b976c47436d98cbcaf/doc/user/README.md#to-generate-diagrams

Wanna double check that it works for you too? Maybe it's a browser/platform thing?

Reply:
I'll be damned. Works for me. Sorry I didn't actually look!

Reply:
No no, glad it was that easy!